### ISSUE TITLE
Broken repo link on 'Before running the sample for the first time'

### ISSUE THREAD

**Issue by rdweaver** [Azure-Samples/communication-services-web-calling-hero#2]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
When you follow the steps on the 'Before running the sample for the first time' on: 
#2. git clone https://github.com/Azure/Communication.git
That git repo, no longer contains the calling samples as of Oct 22, 2020. 
### Any log messages given by the failure
No error message, just the calling samples are missing.

### Expected/desired behavior
I expect the calling sample to be included when I clone that repo

### OS and Version?
not applicable
### Versions
not applicable

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by alex-kwan** [comment]
Sorry about that. They have been since migrated to this location. 

https://github.com/Azure-Samples/communication-services-web-calling-hero

I'll be working to fix up the documentation. Thanks for raising this issue

**Comment by alex-kwan** [comment]
Should be solved with this 14a0cd372077a50c77ff667f9e41e7b12fb56f86
I believe this should successfully solve this issue?
